packages/paste-core/components/base-radio-checkbox/__tests__/base-radio-checkbox.test.tsx

```diff
@@ -115,8 +121,12 @@ describe('Base radio checkbox customization', () => {
         </BaseRadioCheckboxHelpText>
       </CustomizationProvider>
     );
-    expect(screen.getByTestId('control')).toHaveStyleRule('background-color', 'rgb(214,31,31)');
-    expect(screen.getByTestId('label_text')).toHaveStyleRule('color', 'rgb(14,124,58)');
-    expect(screen.container.querySelector('[data-paste-element="fum"]')).toHaveStyleRule('margin-left', '2rem');
+    // matcher selects the last style rule in the stylesheet, not the last style rule that is currently applied
```

Comment:
For more information, [see previous comment](https://github.com/twilio-labs/paste/pull/2376/files#r862247245) about changes to the style rule matcher.